client/src/app/editor/file-outputs/file-outputs.component.html

```diff
@@ -31,25 +31,24 @@ <h3 mat-subheader>Output Files</h3>
           <ng-container matColumnDef="actions">
             <mat-header-cell *matHeaderCellDef></mat-header-cell>
             <mat-cell *matCellDef="let row">
-              <button mat-icon-button color="primary" matTooltip="Preview" (click)="openPreview(row)"
-                [disabled]="!isImage(row.name)" title="Preview file"><mat-icon>visibility</mat-icon>
+              <button mat-icon-button color="primary" matTooltip="Preview" (click)="openPreview(row)" [disabled]="!isImage(row.name)" title="Preview file">
+                <mat-icon>visibility</mat-icon>
               </button>
 
-              <button mat-icon-button color="primary" matTooltip="Copy API Link"
-                ngxClipboard [cbContent]="getApiLink(row)"
-                (cbOnSuccess)="copyDone()" (cbOnError)="copyDone(true)"
-                title="Copy API link"><mat-icon>link</mat-icon>
+              <button mat-icon-button color="primary" matTooltip="Copy API Link" ngxClipboard [cbContent]="getApiLink(row)" (cbOnSuccess)="copyDone()"
+                (cbOnError)="copyDone(true)" title="Copy API link">
+                <mat-icon>link</mat-icon>
               </button>
 
-              <a mat-icon-button color="primary" matTooltip="Download" [attr.download]="row.name"
-                target="_blank" [href]="row.download_url"
-                title="Download file"><mat-icon>file_download</mat-icon>
+              <a mat-icon-button color="primary" matTooltip="Download" [attr.download]="row.name" target="_blank" [href]="row.download_url"
+                title="Download file">
+                <mat-icon>file_download</mat-icon>
               </a>
             </mat-cell>
           </ng-container>
 
           <mat-header-row *matHeaderRowDef="displayedColumns"></mat-header-row>
-          <mat-row *matRowDef="let row; columns: displayedColumns;"></mat-row>
+          <mat-row *matRowDef="let row; columns: displayedColumns;" [@staggerIn]="'showing'"></mat-row>
```

Comment:
Instead you can write `@staggerIn`. The property binding here is not really necessary because the value of `showing` is always static anyways. Then I'd rather query the mat-rows and apply the animation on the `mat-table` element. This way we can really stagger the row elements and it looks really nice when they enter. For the animation see my other comment on your animation code.